app/helpers/sharing_permissions_helper.rb

```diff
@@ -1,37 +1,19 @@
 module SharingPermissionsHelper
 
-  def get_permission_list (item)
-
-    item_id = "#{item.class.name}_#{item.id}"
-
-    policy = []
-
-    # policy
-    downloadable = item.try(:is_downloadable?)
-    policy_text = "<strong>Public:</strong> #{Policy.get_access_type_wording(item.policy.access_type, downloadable)}."
-    sharing_policy_changed = (@batch_sharing_permission_changed && (@items_for_sharing.include? item) && !@policy_params[:access_type].nil?)? "sharing_permission_changed" : ""
-
-    policy.append("<div id=\"#{item_id}-Permission-Policy\" class=\"permission-node #{sharing_policy_changed}\">#{policy_text}</div>".html_safe)
-
-    #permission
+  def get_permission_type (permission)
     option = { target: :_blank }
-    item.policy.permissions.map do |permission|
-      sharing_policy_changed = @batch_sharing_permission_changed && (@items_for_sharing.include? item) ? PolicyHelper::permission_changed_item_class(permission, @policy_params) : ""
-      m = permission.contributor
-      div_id = "#{item_id}-Permission-#{permission.contributor_type}-#{m.id}:"
-      policy_text = "<strong>#{permission.contributor_type}  #{link_to(h(m.title), m, option)}:</strong>"
-      case permission.contributor_type
-      when Permission::WORKGROUP
-        institution = Institution.find(m.institution_id)
-        project = Project.find(m.project_id)
-        policy_text = "<strong> #{link_to(h(project.title), project, option)}  @  #{link_to(h(institution.title), institution, option)}:</strong>"
-      when Permission::PERSON
-        policy_text = "<strong>#{link_to(h(m.title), m, option)}:</strong>"
-      end
-      policy_text += " #{Policy.get_access_type_wording(permission.access_type, downloadable)}."
-      policy.append("<div id=\"#{div_id}\" class=\"permission-node #{sharing_policy_changed}\">#{policy_text}</div>".html_safe)
+    m = permission.contributor
+    case permission.contributor_type
+    when Permission::WORKGROUP
+      institution = Institution.find(m.institution_id)
+      project = Project.find(m.project_id)
+      p_type = "#{link_to(h(project.title), project, option)}  @  #{link_to(h(institution.title), institution, option)}"
+    when Permission::PERSON
+      p_type = "#{link_to(h(m.title), m, option)}"
+    else
+      p_type = "#{permission.contributor_type}  #{link_to(h(m.title), m, option)}"
     end
-    safe_join(policy)
+    p_type.html_safe
```

Comment:
This method mostly duplicates this one:
https://github.com/seek4science/seek/blob/00d8027110f7a1553768fa2de645f509e442ec08/app/helpers/policy_helper.rb#L162

(except with links)

Maybe consider re-using the other method, perhaps with an option to add the links (if that's important)

Reply:
Replaced in [3336fbf](https://github.com/seek4science/seek/pull/1551/commits/3336fbf1bef2add125b7da127d63bc5cc458e96c). I added the link option, because the links were there before my changes... but they can be turned off.